execution-environment.yml

```diff
@@ -51,6 +52,7 @@ dependencies:
     python-daemon
     pyyaml
     six
+    google-cloud-storage
```

Comment:
Which collection depends on this? We should avoid adding deps directly to awx-ee whenever possible.

Reply:
Removed the google-cloud-storage python module to keep things light. This is now just adding ansible.utils.